src/web/server.ts

```diff
@@ -0,0 +1,29 @@
+import express, { Request, Response } from 'express';
```

Comment:
You don't need to do this yourself, use an npm module like [`http-server`](https://www.npmjs.com/package/http-server) instead.